packages/ui/package.json

```diff
@@ -0,0 +1,115 @@
+    "ui",
+    "design-system",
+    "front-end",
+    "web",
+    "react",
+    "react-component",
+    "aws-northstar"
+  ],
+  "scripts": {
+    "build:readme": "cp ./README.md ./build/README.md",
+    "build:copy-files": "cp ../../(LICENSE|NOTICE) ./build/",
+    "build:package-json": "(cat package.json | jq 'del(.devDependencies, .resolutions, .scripts, .\"lint-staged\")') > ./build/package.json",
+    "build:package": "tsc --build tsconfig.build.json",
+    "build": "yarn build:package && yarn build:package-json && yarn build:readme && yarn build:copy-files",
+    "check:all": "yarn test:ci && yarn build && yarn storybook:build && yarn styleguide:build",
+    "license:check": "yarn license-checker --summary --production --excludePrivatePackages --onlyAllow 'MIT;Apache-2.0;Unlicense;BSD;BSD*;BSD-2-Clause;BSD-3-Clause;ISC;Zlib;WTFPL;'",
+    "semantic-release": "semantic-release --dry-run",
+    "storybook:build": "build-storybook -o storybook.out",
+    "storybook": "start-storybook -p 6006",
+    "test:ci": "yarn test --coverage --silent",
+    "test": "jest",
+    "test:watch": "jest --runInBand --watch ./src"
+  },
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws/aws-northstar"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.2",
+    "@cloudscape-design/jest-preset": "^2.0.6",
+    "@storybook/addon-a11y": "^6.5.14",
+    "@storybook/addon-actions": "^6.5.14",
+    "@storybook/addon-essentials": "^6.5.14",
+    "@storybook/addon-interactions": "^6.5.14",
+    "@storybook/addon-links": "^6.5.14",
+    "@storybook/builder-webpack5": "^6.5.14",
+    "@storybook/jest": "^0.0.10",
+    "@storybook/manager-webpack5": "^6.5.14",
+    "@storybook/react": "^6.5.14",
+    "@storybook/testing-library": "^0.0.13",
+    "@storybook/testing-react": "^1.3.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest-axe": "^3.5.3",
+    "@types/lodash.orderby": "^4.6.7",
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "@types/react-router-dom": "^5",
+    "babel-jest": "^29.3.1",
+    "babel-loader": "^8.3.0",
+    "babel-preset-jest": "^29.2.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "29.3.1",
+    "jest-axe": "^7.0.0",
+    "jest-environment-jsdom": "^29.3.0",
+    "license-checker": "^25.0.1",
+    "lodash.orderby": "^4.6.0",
+    "merge": "^2.1.1",
+    "react": "^18",
+    "react-dom": "^18",
+    "react-markdown": "^8.0.5",
+    "react-router-dom": "^6",
+    "react-syntax-highlighter": "^15.5.0",
+    "storybook-addon-react-router-v6": "^0.2.1",
+    "ts-jest": "^29.0.3",
+    "typescript": "^4.9.3"
+  },
+  "dependencies": {
+    "@cloudscape-design/collection-hooks": "^1.0.19",
+    "@cloudscape-design/components": "^3.0.112",
+    "@cloudscape-design/design-tokens": "^3.0.7",
```

Comment:
Possibly more a feature request but I wonder how theming might work with northstar?

Cloudscape themes are produced via code generation and you import the generated components instead of `@cloudscape-design/components` (see https://cloudscape.design/foundation/visual-foundation/theming/#implementation) - might need to somehow provide northstar references to the components it should be building upon in order for a custom theme to apply.

Reply:
Thanks for the suggestions.